### DISCUSSION TITLE
LLDPD --with-snmp support.

### DISCUSSION THREAD

**Discussion by ndivakaran** [lldpd/lldpd#514] [Q&A]
Hi I need to cross compile lldpd with snmp support.But after compile log shows Optional features:
  SNMP support...: no
  CDP............: yes
  FDP............: yes
  EDP............: yes
  SONMP..........: yes
  LLDPMED........: yes
  DOT1...........: yes
  DOT3...........: yes
  CUSTOM.........: yes
  XML output.....: no
  Oldies support.: no
  seccomp........: no
  libbsd.........: no

My script file ilike 

./configure --prefix=$lldpInstallDir --host=$toolchain --target=$toolchain --with-systemdsystemunitdir=$lldpInstallDir --with-sysusersdir=$lldpInstallDir --with-snmp=$netsnmpInstallDir/bin --with-lldpd-ctl-socket=/var/run/lldpd.socket --with-privsep-user=$privsepUser --with-privsep-group=$privsepGroup

**Comment by EnzoQiu** [comment]
"--with-snmp" is enough.
